content/en/blog/pop-joins-falco.md

```diff
@@ -0,0 +1,58 @@
+---
+ title: POP joins Falco
+ date: 2020-09-12
+ author: Dan "POP" Papandrea
+ ---
+
+ ## Join POP Falco.org
+
+ ***Me***
+
+
+ My name is Dan “POP” Papandrea and I am the new Lead of Open Source Community and Ecosystem at Sysdig, with a focus on Falco adoption and contribution. Throughout my career, I have focused on trying to walk a mile in end users' shoes to understand their software pains. Over the past four years at Sysdig and during my time in the cloud-native community, I have learned so much about how enterprises and their end users, developers and operators are using cloud-native applications to truly transform their business and speed their delivery.  I have also immersed myself in the open source community and seen this amazing group blossom into a fantastic place for innovation and collaboration.
+
+ ***So, Why the Move to Falco?***
+
+ Runtime security is clearly a necessary requirement for protecting cloud native, and specifically Kubernetes, and there are amazing opportunities as organizations transition to securing container and Kubernetes environments. Falco is cementing itself as the runtime security tool for the cloud.
+
+ Having worked with the Falco project since its inception in my capacity as Field CTO for Sysdig, I have seen Falco’s incredible momentum and adoption rate.  Falco is a phenomenal piece of core technology, and it has been exciting to watch the maintainers who contribute to the project create even more exciting things.  Getting to work with the Falco community and my colleagues at Sysdig involved in Falco development has been such a pleasure.
+
+ I chose to move from my Field CTO position to Falco because I believe in the technology and I love the cloud-native community.  I started a [podcast](https://github.com/danpopSD/popcast#show-follow-and-subscribe-to-our-youtube-channel--twitter) the popcast with danpop at the beginning of this pandemic as a love letter to the community and to connect at a human level with the leaders who created its awesome tech, beyond just their code.  I am a member of the Kubernetes Contributor group, who eats, sleeps and breathes how to best contribute, adopt, and support what I believe is the greatest technical community in the world -- Kubernetes -- and its constellation of supporting projects.  Being able to work with the amazing ecosystem within the CNCF and the various surrounding projects is so inspiring and I cannot wait to help nurture adoption, contribution, and end-user success.  This is what gets me pumped.  I am such a big fan of these projects and there is so much power and possibility if we work together in the true spirit of collaboration and COMMUNITY.
+
+ ***Falco’s Progress***
```

Comment:
this whole section is just vanity metrics

what has falco done? what features have been impactful? 

you mention a lot of contributors/maintainers but what have they done?

- Jonah enabled our builds and hosting our drivers
- Spencer has been the biggest voice for the helm chart
- Leo and Lore have built a state of the art build system from the ground up
- Leo has basically taken over the helm chart and has single handily debugged the drops using his event generator framework